Flow.Launcher.Core/Plugin/JsonRPCModelContext.cs

```diff
@@ -0,0 +1,21 @@
+﻿using System.Text.Json.Serialization;
+
+namespace Flow.Launcher.Core.Plugin
+{
+    // TODO: After Upgrading to .Net 7, adding Source Generating Context for IAsyncEnumerable JsonRPCMessage
```

Comment:
We are already on .Net 7

Reply:
oops that's an old one, we should remove it